client/src/styles/login-form.scss

```diff
@@ -207,7 +207,23 @@ input[type="checkbox"] {
   }
 }
 
-.popover .popover-header + .popover-body {
-  padding: 12px 15px;
-  font-size: 13px;
+.popover {
+  background-color: var(--color-form-background);
+  border-color: var(--color-field-border);
+
+  &.bs-popover-auto[x-placement^="top"] > .arrow::before,
```

Comment:
In some cases the popover could be placed at the bottom 

![image](https://user-images.githubusercontent.com/10215604/122839833-f0735e80-d34c-11eb-85ec-d3ecc40f5348.png)

Reply:
This should fix it https://github.com/silverstripe/silverstripe-login-forms/pull/98/commits/acc1b4b4972c44c5bce91f5f7143e0057f0e2a37